azurelinuxagent/common/protocol/wire.py

```diff
@@ -51,6 +51,8 @@
 
 WIRE_SERVER_ADDR_FILE_NAME = "WireServer"
 INCARNATION_FILE_NAME = "Incarnation"
+SEQUENCE_NUMBER_FILE_NAME = "SequenceNumber"
```

Comment:
Since the extensions also have a `SequenceNumber`, could you rename this to something like `artifactProfileSeqNo` or something similar to avoid any unnecessary ambiguity